definitions/npm/react-select_v1.x.x/flow_v0.53.x-/react-select_v1.x.x.js

```diff
@@ -1,13 +1,6 @@
 declare module 'react-select' {
-  declare type SelectOption = {
-    value: string | number,
-    label: string,
-    clearableValue?: boolean,
-  };
-
-  declare type Option = {
-    [key: string]: any,
-  };
+  declare type OptionType = Object
```

Comment:
But between `Object` and `{ [key: string]: any }` there's not much differences? 
Except if one need fancy keys.

Reply:
Good observation, actually, I have a case when { [key: string]: any } fails, when options are a Union type. I'll modify the test